lab_5_scrapper/scrapper.py

```diff
@@ -221,25 +221,24 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
             url = el.get('href')
             if url.split('/')[2] == 'servernews.ru':
                 continue
+            url = self.url_pattern + url
             if url not in self.urls:
                 break
-        url = self.url_pattern + url
         return url
 
     def find_articles(self) -> None:
         """
         Find articles.
         """
         seeds = self.get_search_urls()
-        for seed in seeds:
-            response = make_request(seed, self.config)
-            if not response.ok:
-                continue
-            article_bs = BeautifulSoup(response.text, 'html.parser')
-            urls = []
-            for i in range(25):
-                urls.append(self._extract_url(article_bs))
-            self.urls.extend(urls)
+        while len(self.urls) < self.config.get_num_articles():
```

Comment:
yeah, i think while is more reasonable here

Reply:
ok